sdk/edgezones/arm-edgezones/review/arm-edgezones.api.md

```diff
@@ -0,0 +1,212 @@
+## API Report File for "@azure/arm-edgezones"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ClientOptions } from '@azure-rest/core-client';
+import { OperationOptions } from '@azure-rest/core-client';
+import { Pipeline } from '@azure/core-rest-pipeline';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export type ActionType = string;
+
+// @public
+export type ContinuablePage<TElement, TPage = TElement[]> = TPage & {
+    continuationToken?: string;
+};
+
+// @public
+export type CreatedByType = string;
+
+// @public (undocumented)
+export class EdgeZonesClient {
+    constructor(credential: TokenCredential, subscriptionId: string, options?: EdgeZonesClientOptionalParams);
+    readonly extendedZones: ExtendedZonesOperations;
+    readonly operations: OperationsOperations;
+    readonly pipeline: Pipeline;
+}
+
+// @public
+export interface EdgeZonesClientOptionalParams extends ClientOptions {
+    apiVersion?: string;
+}
+
+// @public
+export interface ErrorAdditionalInfo {
+    readonly info?: Record<string, any>;
```

Comment:
could you link the spec with me?

Reply:
this model is from common-type
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/types.json#L291-L299